BDArmory/Distribution/GameData/BDArmory/Localization/UI/ja.cfg

```diff
@@ -0,0 +1,1194 @@
+        #LOC_BDArmory_VesselMover_Help_AutoRotateRocket = ロケットの自動回転
+        #LOC_BDArmory_VesselMover_Help_AutoRotatePlane = 飛行機の自動回転
+        #LOC_BDArmory_VesselMover_Help_CycleAltitudes = サイクルプリセット高度: Tab、Shift+Tab
+        #LOC_BDArmory_VesselMover_Help_ResetAltitude = 高度をリセット
+        #LOC_BDArmory_VesselMover_Help_AdjustAltitude = 高度を調整
+        #LOC_BDArmory_VesselMover_CloseOnCompetitionStart = 対戦開始時に閉じる
+
+        // Craft Browser
+        #LOC_BDArmory_CraftBrowser_InvalidParts = 無効な部品
+        #LOC_BDArmory_CraftBrowser_UnknownModules = モジュール
+        #LOC_BDArmory_CraftBrowser_Clear = クリア
+        #LOC_BDArmory_CraftBrowser_ClearAll = 全てクリア
+        #LOC_BDArmory_CraftBrowser_Refresh = 更新
+        #LOC_BDArmory_CraftBrowser_Parts = 部品
+        #LOC_BDArmory_CraftBrowser_Mass = 質量
+        #LOC_BDArmory_CraftBrowser_Version = バージョン
+        #LOC_BDArmory_CraftBrowser_Craft = クラフト
+        #LOC_BDArmory_CraftBrowser_Folder = フォルダー
+
+        // Waypoints game mode
+        #LOC_BDArmory_Settings_WaypointsMode =ウェイポイントモード
+        #LOC_BDArmory_Settings_WaypointsOptions = ウェイポイントの設定
+        #LOC_BDArmory_Settings_WaypointsOneAtATime = 一つずつ
+        #LOC_BDArmory_Settings_WaypointsInfFuelAtStart = 最初のウェイポイントまで無限燃料
+        #LOC_BDArmory_Settings_WaypointsShow = ウェイポイントを表示
+
+        // Tournaments
+        #LOC_BDArmory_Settings_TournamentOptions = トーナメント設定
+        #LOC_BDArmory_Settings_TournamentStyle = トーナメントの種類
+        #LOC_BDArmory_Settings_TournamentRoundType = ラウンドの種類
+        #LOC_BDArmory_Settings_TournamentDelayBetweenHeats = リスポーン間の遅延
+        #LOC_BDArmory_Settings_TournamentTimeWarpBetweenRounds = ラウンド間のタイムワープ
+        #LOC_BDArmory_Settings_TournamentRounds = ラウンド
+        #LOC_BDArmory_Settings_TournamentVesselsPerHeat = 一回当たりの機体数
+        #LOC_BDArmory_Settings_TournamentVesselsPerTeam = 一回当たりのチームの機体数
+        #LOC_BDArmory_Settings_TournamentTeamsPerHeat = 一回あたりのチーム数
+        #LOC_BDArmory_Settings_GauntletOpponentsFilesLocation = 対戦相手のファイル
+        #LOC_BDArmory_Settings_TournamentOpponentTeamsPerHeat = 一回当たりの敵チーム
+        #LOC_BDArmory_Settings_TournamentOpponentVesselsPerTeam = チーム当たりの対戦相手の機体
+        #LOC_BDArmory_Settings_TournamentFullTeams = クラフトを再利用してチームを埋める
+        #LOC_BDArmory_Settings_TournamentNPCsPerHeat = 体力ごとのNPC数
+        #LOC_BDArmory_Settings_TournamentSetup = トーナメントのセットアップ
+        #LOC_BDArmory_Settings_TournamentRun = トーナメントを開始
+        #LOC_BDArmory_Settings_TournamentStop = トーナメントを終了
+
+        // Scores
+        #LOC_BDArmory_BDAScores_Title = トーナメントスコア
+        #LOC_BDArmory_BDAScores_Weights = スコアの量
+        #LOC_BDArmory_BDAScores_Round = ラウンド
+        #LOC_BDArmory_BDAScores_Heat = 体力
+
+        // Custom Spawn Templates
+        #LOC_BDArmory_Settings_CustomSpawnTemplateOptions = スポーンテンプレート設定
+        #LOC_BDArmory_Settings_SpawnOnly = スポーンのみ
+        #LOC_BDArmory_Settings_SpawnAndStartCompetition = スポーンして対戦を開始
+        #LOC_BDArmory_Settings_CustomSpawnTemplate_Refresh = リロード

```

Comment:
`#LOC_BDArmory_Settings_CustomSpawnTemplate_Refresh` isn't used anymore, `#LOC_BDArmory_CraftBrowser_Refresh` is used instead.

`#LOC_BDArmory_Settings_CustomSpawnTemplate_ReplaceTeam = Replace Teams` is missing, which is used to replace all the slots on a team in a custom spawn template with the selected craft.